packages/__docs__/package.json

```diff
@@ -124,7 +124,7 @@
     "html-webpack-plugin": "^4",
     "jsdoc-api": "^6.0.0",
     "mkdirp": "^1",
-    "react-docgen": "^5",
+    "react-docgen": "^6.0.0-alpha.0",
```

Comment:
This new importer feature is not fully released in production yet, but according to the maintainers of the repo v6 won't contain any breaking changes, so it seems like a safe thing for us to use the pre-release version of the lib.

Reply:
The yarn.lock is missing, please add it :) I couldn't find any problems with it, the View props display correctly now, and it could recognise the "query" and "props" prop of Responsive as "object" (typeof this and that) too. Well, it cannot go deeper than one level, but that's why we need that type-linking ticket too :)